lib/create-app.js

```diff
@@ -25,6 +25,7 @@ const path = require('path')
 const { routeResolvedFile } = require('./utils')
 const ResourceMapper = require('./resource-mapper')
 const aclCheck = require('@solid/acl-check')
+const version = require('../package.json').version
```

Comment:
You might want to `const { version } = require('../package.json')`.

Reply:
OK, done!